### PR TITLE
added *args to `inventory_data_to_pandas` to fetch desired system facts.

### PR DESCRIPTION
**Pull request by parsahosseini**
* we can now specify what system facts we wish to isolate by providing their names in `inventory_data_to_pandas`.
* `fetch_s3` is removed since this will no longer be our desired data-source.

### CODE REVIEW

File: rad/rad.py [hunk 1]
```diff
@@ -167,6 +131,11 @@ def inventory_data_to_pandas(dic):
         row = {"id": system_id}
 
         for key, value in system_profile.items():
+            unique_keys.add(key)
+
+            # ensure the system profile is the one desired
+            if key not in args and len(args) != 0:
```
Comment (jameswnl) [inline rad/rad.py]: you could do `if not args and key not in args:`
that would short-circuit the 2nd condition (which is much costlier) if `args` is an empty list

**Comment by jameswnl** [comment]
@parsahosseini so this won't narrow down those entries that itself contains lot of columns, right? e.g. the `installed_packages`.

File: rad/rad.py (hunk 1 above)
Comment (parsahosseini) [reply rad/rad.py]: @jameswnl will do - good catch.

**Comment by parsahosseini** [comment]
@jameswnl if you provide arguments, yes... it narrows it down to only those arguments. If you omit features which cause lots of columns (i.e. `installed_packages`), then those columns are gone / no longer there.

**Pull request merged**